### ISSUE TITLE
1.2 Flashing Failed

### ISSUE THREAD

**Issue by nitz** [pine64/pinecil-firmware-updater#4]
Hello!

Was giving a shot updating today, figured I'd try this new tool out for a whirl. Gave it a shot updating to 2.14.1 (English), and it failed to flash.

I went back to the GD32 DFU tool and was able to update via it, at least!

Here's the full output:

![image](https://user-images.githubusercontent.com/602691/105748984-d144e200-5f10-11eb-902d-516634719551.png)

**Comment by gamelaster** [comment]
Hi @nitz, it doesn't see Pinecil connected for some reason.
Restarting the app didn't helped?

**Comment by nitz** [comment]
Hm, I might have taken that screenshot after I disconnected! 

Here's what it looks like leaving it connected:

![image](https://user-images.githubusercontent.com/602691/105760419-c47bba80-5f1f-11eb-9d63-e77641b01bb9.png)

**Comment by gamelaster** [comment]
You have probably installed other driver for Pinecil, so zadic don't install Winusb driver. If you installed any driver with ZADIG, uninstall it and it should work.

**Comment by nitz** [comment]
That did it! Well, a bit anyways:

I hadn't used Zadig until literally two days ago, when I was installing a driver for a USB Logic Analyzer. I used Zadig that came with PulseView for that. I'm not *positive* if it has anything to do with anything, as I didn't remove it, but funny none the less.

I didn't install any other driver with ZADIG, but I *had* installed the GD32 DFU Drivers, since that's how I had updated my pinecil before. Uninstalling those specifically let Zadig install  the ones it wanted to for `28E9:0189`, and it worked just fine!

I'm not sure if you want to have a specific message for a case like this one (where it doesn't find the driverless usb device, but tries to flash anyways), or if you figure my edge case is unique enough to write off as a one-off. I'll let you close the issue if you feel so!

Thanks for the help, and for the tool!

**Comment by gamelaster** [comment]
@nitz Glad to hear that! 😊

I'm thinking if zadic (that's command line version of Zadig, what is used in this tool) can be forced to anyway replace driver, if he found other. So I think let's mark this as bug and fix it.

Thank you :)

**Comment by Ralim** [comment]
@gamelaster 
I think the new updater should help with this?

**Comment by gamelaster** [comment]
@Ralim well, @River-b was trying it and as far as I remember, it didn't worked until he removed the GDU driver, even when I tried to force zadic to use All available devices. I will try it once again, but better will be to use new tool, since zadic is depreciated and removed.
Thanks

**Comment by River-b** [comment]
yes I deleted the whole folder I had for the older GDU driver that I was using previously  to install Custom versions of IronOS.
But also there was another problem with blank screen (no photo of pinecil)   and Gamelaster did some other magic as well.
